package.json

```diff
@@ -16,6 +16,7 @@
   "author": "Wild Code School",
   "license": "ISC",
   "devDependencies": {
+    "@types/prop-types": "^15.7.5",
```

Comment:
no need to install prop-types in the root package.json : restore it (and restore the root package-lock.json)